CHANGELOG.md

```diff
@@ -6,6 +6,18 @@
 
 ### Fixes
 
+## v1.23.0 (2022-05-10)
+
+### Enhancements
+
+- Add error icon to errored services in trace list view ([@Jaskerv](https://github.com/Jaskerv) in [#927](https://github.com/jaegertracing/jaeger-ui/pull/927))
+
+### Fixes
```

Comment:
just merged [Fix monitor tab crashing (](https://github.com/jaegertracing/jaeger-ui/commit/0003deb2ca9e32efa2c49db7c8c9aa3b59f44aee)https://github.com/jaegertracing/jaeger-ui/pull/946[)](https://github.com/jaegertracing/jaeger-ui/commit/0003deb2ca9e32efa2c49db7c8c9aa3b59f44aee), can we include it?

Reply:
done, included in the release/changelog